src/PRASBase/assets.jl

```diff
@@ -236,6 +290,70 @@ Base.:(==)(x::T, y::T) where {T <: GeneratorStorages} =
     x.λ == y.λ &&
     x.μ == y.μ
 
+Base.getindex(g_s::G_S, idxs::AbstractVector{Int}) where {G_S <: GeneratorStorages} =
```

Comment:
Could `G_S` just be `G`? I'd prefer to reserve multi-character all caps names for global constants.